cli/ingress-controller/main.go

```diff
@@ -397,12 +402,13 @@ func main() {
 		}()
 	}
 	kong.Start()
+	wg.Wait()
+	os.Exit(<-exitCh)
 }
 
 type exiter func(code int)
 
-func handleSigterm(kong *controller.KongController, stopCh chan struct{},
-	exit exiter) {
+func handleSigterm(kong *controller.KongController, stopCh chan<- struct{}, exit chan<- int) {
```

Comment:
nit: Can we rename the `exit` channel to `exitCh` to make the code easier to follow?